collect_app/src/main/java/org/odk/collect/android/formlists/blankformlist/BlankFormListItemView.kt

```diff
@@ -1,25 +1,18 @@
 package org.odk.collect.android.formlists.blankformlist
 
+import android.content.Context
 import android.view.LayoutInflater
 import android.view.View
-import android.view.ViewGroup
 import android.widget.FrameLayout
-import androidx.recyclerview.widget.RecyclerView
-import org.odk.collect.android.R
 import org.odk.collect.android.databinding.BlankFormListItemBinding
+import org.odk.collect.strings.R.string
 import timber.log.Timber
 import java.text.SimpleDateFormat
 import java.util.Locale
 
-class BlankFormListItemViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(
-    BlankFormListItemBinding.inflate(
-        LayoutInflater.from(parent.context),
-        parent,
-        false
-    ).root
-) {
+class BlankFormListItemView(context: Context) : FrameLayout(context) {
 
-    val binding = BlankFormListItemBinding.bind(this.itemView)
+    val binding = BlankFormListItemBinding.inflate(LayoutInflater.from(context), this, true)
 
     var blankFormListItem: BlankFormListItem? = null
```

Comment:
It feels more "Kotlin" to use a property rather than a method to set the "data" on a view. I do think I prefer custom views having a single setup method though like you've suggested, so I'm glad we agree! I'll switch it.